apps/src/applab/dataBrowser/dataStyles.js

```diff
@@ -15,12 +16,21 @@ export const row = {
 export const cell = {
   padding: cellPadding,
   border: '1px solid gray',
-  fontSize: 14
+  fontSize: 14,
+  maxWidth: 350,
+  overflow: 'hidden',
 };
 
-export const buttonCell = [cell, {
-  width: buttonCellWidth,
+export const addButtonCell = [cell, {
+  minWidth: addButtonCellWidth,
   backgroundColor: color.lightest_teal,
+  width: addButtonCellWidth,
+}];
+
+export const editButtonCell = [cell, {
+  minWidth: editButtonCellWidth,
+  backgroundColor: color.lightest_teal,
+  width: editButtonCellWidth,
```

Comment:
I wonder if it would be cleaner to export objects instead of arrays, this way you don't need to depend on your consuming component to be using Radium, i.e.
```
export const editButtonCell = Object.assign({}, cell, {
  minWidth: editButtonCellWidth,
  backgroundColor: color.lightest_teal,
  width: editButtonCellWidth,
});
```
With object rest destructuring, we could even do
```
export const editButtonCell = {
  ...cell,
  minWidth: editButtonCellWidth,
  backgroundColor: color.lightest_teal,
  width: editButtonCellWidth,
});